### PR TITLE
Add jest-preset-gatsby

### PR DESCRIPTION
**Pull request by keplersj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,6 +110,7 @@
 - [@bigtest/interactor](https://www.bigtestjs.io/guides/interactors/introduction/) Composable Page Objects for component based applications. Checkout the [documentation](https://www.bigtestjs.io/guides/interactors/introduction/) and [blog post](https://medium.com/@robdel12/jest-bigtest-interactor-component-test-%EF%B8%8F-11b1947954c8) about using with Jest.
 - [@testdeck/jest](https://github.com/testdeck/testdeck/) Decorator based wrapper around Jest for object oriented testing. Checkout the [documentation](https://testdeck.org) and a working [seed](https://github.com/testdeck/testdeck-jest-seed) for you to start with.
 - [jest-plugin-must-assert](https://github.com/ballercat/jest-plugin-must-assert) A Jest plugin for strict runtime checks. Fails any tests without assertions and prevents async tasks (Promises & timeouts) from leaking across tests.
+- [jest-preset-gatsby](https://github.com/keplersj/jest-preset-gatsby) Jest preset to streamline unit testing a Gatsby project.
```
Comment (rickhanlonii) [inline README.md]: Should we start a Preset header?

Comment (keplersj) [reply README.md]: Makes sense to me. Definitely think that the preset functionality is super powerful and could benefit from some promotion on here. I'll draft up a PR with the presets on this list, and the presets mentioned in the Jest documentation.

Comment (keplersj) [reply README.md]: Created #71!

**Comment by keplersj** [comment]
Superseded by #71

File: README.md (hunk 1 above)
Comment (rickhanlonii) [reply README.md]: That's great, thanks!

**Pull request closed without merging**